docs/networking/networking-basics.md

```diff
@@ -0,0 +1,520 @@
+# Networking Basics
+
+> Written by Dan Ford and originally published on <a href="https://dlford.io/computer-networking-binary-hex-explained/" target="_blank" rel="noopener noreferrer">dlford.io</a>.<br />
+> Copyright &copy; 2019 dlford.io all rights reserved.
```

Comment:
Sorry I don't think we can take this article into the wiki with the attached copyright

Reply:
Unfortunately, i'm inclined to agree. 

That being said, if you wanted to thin this post down (a la https://github.com/selfhostedshow/wiki/pull/7/files#r382993359), and link through to your article for further details, I think i'd be ok with that?